src/sentry/static/sentry/app/components/events/errorLevel.jsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'react-emotion';
+
+const DEFAULT_SIZE = '13px';
+
+function getLevelColor({level, theme}) {
+  const COLORS = {
```

Comment:
Can you pull the constant up into the module scope

Reply:
it maps level -> theme